### PR TITLE
Jennyf/msal4.8.2

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

File: UserDetailsClient/UserDetailsClient.Core/Features/LogOn/B2CConstants.cs [hunk 1]
```diff
@@ -5,7 +5,7 @@ public static class B2CConstants
         // Azure AD B2C Coordinates
         public static string Tenant = "fabrikamb2c.onmicrosoft.com";
         public static string AzureADB2CHostname = "fabrikamb2c.b2clogin.com";
-        public static string ClientID = "90c0fe63-bcf2-44d5-8fb7-b8bbc0b29dc6";
+        public static string ClientID = "e5737214-6372-472c-a85a-68e8fbe6cf3c";
```
Comment (jmprieur) [inline UserDetailsClient/UserDetailsClient.Core/Features/LogOn/B2CConstants.cs]: why did we change the client ID?

Comment (jennyf19) [reply UserDetailsClient/UserDetailsClient.Core/Features/LogOn/B2CConstants.cs]: @jmprieur the earlier one no longer works. for some reason, the redirect uri for mobile is no longer included, and i couldn't add it (invalid redirecturi error) and it was automatically try to go to localhost, so i just switched to this clientId, in the same tenant, same policies apply.

Comment (jmprieur) [reply UserDetailsClient/UserDetailsClient.Core/Features/LogOn/B2CConstants.cs]: Thanks for the explanations, @jennyf19

**Comment by jmprieur** [comment]
Tested on Android and UWP

**Pull request merged**